model_analyzer/reports/report_manager.py

```diff
@@ -231,6 +245,51 @@ def _build_summary_table(self, report_key, num_measurements, gpu_name):
             summary_table.insert_row_by_index(row)
         return summary_table, summary_sentence
 
+    def _build_summary_cpu_table(self, report_key, num_measurements):
+        """
```

Comment:
This function seems to have a lot in common with `_build_summary_table`. Would it be possible to simply add an extra argument like `cpu_only` or soemthing to account for the modified functionality?

Reply:
Sure, will fix it in the review.